spec.html

```diff
@@ -48221,10 +48237,12 @@ <h1>
           1. Let _generator_ be the value of the Generator component of _genContext_.
           1. Assert: GetGeneratorKind() is ~sync~.
           1. Set _generator_.[[GeneratorState]] to ~suspended-yield~.
-          1. Remove _genContext_ from the execution context stack and restore the execution context that is at the top of the execution context stack as the running execution context.
+          1. Remove _genContext_ from the execution context stack.
           1. Let _callerContext_ be the running execution context.
-          1. Resume _callerContext_ passing NormalCompletion(_iteratorResult_). If _genContext_ is ever resumed again, let _resumptionValue_ be the Completion Record with which it is resumed.
-          1. Assert: If control reaches here, then _genContext_ is the running execution context again.
+          1. Transfer control from _genContext_ to _callerContext_, passing NormalCompletion(_iteratorResult_).
+          1. NOTE: This step is reached only if/when control is transferred back to _genContext_.
```

Comment:
I like the wording you used above.

```suggestion
          1. NOTE: This step is reached only when control is transferred back to _genContext_, which might not happen.
```

Reply:
Done (x9), though I still prefer the "if/when" phrasing.